crates/router/src/core/payments/operations/payment_create_intent.rs

```diff
@@ -0,0 +1,438 @@
+            enable_payment_link: self.payment_link_enabled.clone().unwrap_or_default(),
+            apply_mit_exemption: self.apply_mit_exemption.clone().unwrap_or_default(),
+            customer_present: self.customer_present.clone().unwrap_or_default(),
+            payment_link_config: self
+                .payment_link_config
+                .clone()
+                .map(ForeignFrom::foreign_from),
+            routing_algorithm_id: self.routing_algorithm_id.clone(),
+        })
+    }
+
+    fn get_request_incremental_authorization_value(
+        &self,
+    ) -> RouterResult<common_enums::RequestIncrementalAuthorization> {
+        self.request_incremental_authorization
+            .map(|request_incremental_authorization| {
+                if request_incremental_authorization == common_enums::RequestIncrementalAuthorization::True {
+                    if self.capture_method == Some(common_enums::CaptureMethod::Automatic) {
+                        Err(errors::ApiErrorResponse::InvalidRequestData { message: "incremental authorization is not supported when capture_method is automatic".to_owned() })?
+                    }
+                    Ok(common_enums::RequestIncrementalAuthorization::True)
+                } else {
+                    Ok(common_enums::RequestIncrementalAuthorization::False)
+                }
+            })
+            .unwrap_or(Ok(common_enums::RequestIncrementalAuthorization::default()))
+    }
+}
+
+type PaymentsCreateIntentOperation<'b, F> =
+    BoxedOperation<'b, F, PaymentsCreateIntentRequest, payments::PaymentIntentData<F>>;
+
+#[async_trait]
+impl<F: Send + Clone> GetTracker<F, payments::PaymentIntentData<F>, PaymentsCreateIntentRequest>
+    for PaymentCreateIntent
+{
+    #[instrument(skip_all)]
+    async fn get_trackers<'a>(
+        &'a self,
+        state: &'a SessionState,
+        payment_id: &common_utils::id_type::GlobalPaymentId,
+        request: &PaymentsCreateIntentRequest,
+        merchant_account: &domain::MerchantAccount,
+        key_store: &domain::MerchantKeyStore,
+        _auth_flow: services::AuthFlow,
+        _header_payload: &api::HeaderPayload,
+    ) -> RouterResult<
+        operations::GetTrackerResponse<
+            'a,
+            F,
+            PaymentsCreateIntentRequest,
+            payments::PaymentIntentData<F>,
+        >,
+    > {
+        let db = &*state.store;
+        let key_manager_state = &state.into();
+
+        let storage_scheme = merchant_account.storage_scheme;
+
+        let profile_id = request.profile_id.clone();
+
+        let profile = db
+            .find_business_profile_by_profile_id(&(state).into(), key_store, &profile_id)
+            .await
+            .to_not_found_response(errors::ApiErrorResponse::ProfileNotFound {
+                id: profile_id.get_string_repr().to_owned(),
+            })?;
+
+        let payment_intent_domain = request
+            .create_domain_model_from_request(
```

Comment:
We don't need a bridge implementation here, just we can write the implementation directly here.